theme/tests/multiplatform.py

```diff
@@ -202,3 +203,30 @@ def test_create_resource(self):
             self.assertEqual(resource.title, resource_title)
             self.assertEqual(resource.short_id, shortkey)
             self.assertEqual(resource.creator, self.user)
+
+        def test_create_comment(self):
```

Comment:
I added a test for including HTML in comments and user names. Let me know what you think or feel free to add-on to this.